src/main/java/org/sonar/plugins/ldap/windows/auth/WindowsAuthSettings.java

```diff
@@ -118,5 +124,12 @@ public String getGroupIdAttribute() {
   public String getProtocols() {
     return StringUtils.defaultIfBlank(settings.getString(LDAP_WINDOWS_AUTH_SSO_PROTOCOLS), DEFAULT_SONAR_WINDOWS_AUTH_SSO_PROTOCOLS);
   }
+  
+  /**
+   * Returns the specified value for the real name attribute. By default, it's value is "cn"
+   */
+  public String getLdapUserRealNameAttribute(){ 
```

Comment:
Please update unit tests WindowsAuthSettingsTest to  test newly added String fields as well as the method getLdapUserRealNameAttribute().